hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/read/TestECBlockInputStream.java

```diff
@@ -416,6 +416,7 @@ private void validateBufferContents(ByteBuffer buf, int from, int to,
       return blockStreams;
     }
 
+    // Not a thread safe create.
```

Comment:
Should we just make this factory's methods synchronized too? I don't think it can do any harm, and it may avoid some hard to track down bug in the future.